### ISSUE TITLE
Error in SlowRequest , $slowRequest->action become stdClass 

### ISSUE THREAD

**Issue by lam0819** [laravel/pulse#249]
### Pulse Version

^1.0@beta

### Laravel Version

10.10

### PHP Version

8.2

### Livewire Version

^3.0

### Database Driver & Version

MySQL 8

### Description

I am not sure it is my issue or global issue.

I go following error 

`
local.ERROR: htmlspecialchars(): Argument #1 ($string) must be of type string, stdClass given {"view":{"view":"/var/www/html/vendor/laravel/pulse/resources/views/livewire/slow-requests.blade.php","data":[]},"userId":3,"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): htmlspecialchars(): Argument #1 ($string) must be of type string, stdClass given at /var/www/html/vendor/laravel/framework/src/Illuminate/Support/helpers.php:124)
`

finally i found that in slow-requests.blade.php

`
$slowRequest->action
`
become stdClass instead of String.

### Steps To Reproduce

Just a normal laravel , volt and filament projects.

here is some data i come up with in slow_request.
![image](https://github.com/laravel/pulse/assets/68211972/a85ad582-1561-494e-944f-0d23a67e7ad3)

**Comment by crynobone** [comment]
Can you share the route code (typically in `web.php` for `GET /en/page/{slug}`?

**Comment by jessarcher** [comment]
This is apparently caused by a bug in the framework that has been fixed in `v10.38.1` (see https://github.com/laravel/framework/commit/b0ed9e1a601d8cbf122c815198d86b333a881f11)

Unfortunately, this won't fix the existing entries in your Pulse tables. The simplest solution would be to clear the slow request data once you've updated your Laravel version:

```sh
php artisan pulse:clear --type slow_request
```

**Comment by jessarcher** [comment]
Also, great job on debugging where the error came from and sharing that screenshot of your entries table! It was very helpful :slightly_smiling_face:

**Comment by lam0819** [comment]
> This is apparently caused by a bug in the framework that has been fixed in `v10.38.1` (see [laravel/framework@b0ed9e1](https://github.com/laravel/framework/commit/b0ed9e1a601d8cbf122c815198d86b333a881f11))
> 
> Unfortunately, this won't fix the existing entries in your Pulse tables. The simplest solution would be to clear the slow request data once you've updated your Laravel version:
> 
> ```shell
> php artisan pulse:clear --type slow_request
> ```

php artisan pulse:clear --type slow_request

   ERROR  Command "pulse:clear" is not defined. Did you mean one of these?
   
 is this new function ?
 
 
 But i am confirm it is fixed after using v10.38.1

**Comment by timacdonald** [comment]
Ahh, we renamed this in a recent release. Your version likely has the old name

```
php artisan pulse:purge --type slow_request
```
